.travis.yml

```diff
@@ -64,5 +64,9 @@ script:
   # Run pum's test and upgrade
   - yes | pum test-and-upgrade -pp qgep_prod -pt qgep_test -pc qgep_comp -t qgep_sys.pum_info -d $TRAVIS_BUILD_DIR/delta/ -f /tmp/qwat_dump -i constraints views --exclude-schema public -v int SRID 2056
 
+  # Run tests on qgep_prod
+  - export PGSERVICE=qgep_prod
+  - nosetests test/test_import.py
```

Comment:
I'd be in favor of running all the tests on the demo dump too. Or did you try and did not work? (conflict with existing data?)

Reply:
No conflict. But on the test `test_vw_qgep_reach` of `test_views.py` it makes an update what takes more than 10 Minutes with data. So Travis times out: 
```
$ nosetests
............
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
```
Locally it takes that long as well. It's still running...

Question is, if we should optimize the rule to increase the performance for that action or if this test is an "unrealistic" use case, we could skip at least in the test with data.

What do you think?

Reply:
yes, just skip that test for now